engine/lib/views.php

```diff
@@ -1624,6 +1624,10 @@ function elgg_views_boot() {
 		'deps' => array('jquery'),
 		'exports' => 'jQuery.fn.ajaxForm',
 	));
+	elgg_define_js('jquery.ui', array(
+		'src' => '/vendors/jquery/jquery-ui-1.10.4.min.js',
+		'deps' => array('jquery'),
```

Comment:
Is there an "exports" value you can add here? Or does jquery-ui already declare itself as an AMD module?